src/ts/uimanager.ts

```diff
@@ -302,6 +318,22 @@ export class UIManager {
     resolveUiVariant(null);
   }
 
+  /**
+   * Exposes i18n.getLocalizableCallback() function
+   * @returns {I18nApi.getLocalizableCallback()}
+   */
+  static localize<V extends CustomVocabulary<Record<string, string>>>(key: keyof V) {
```

Comment:
If this is simply exposing a method from another class, why is it called totally different? E.g. for `i18n.getLocalizableCallback` I would expect this to be called `getLocalizableCallback` or `getI18nLocalizableCallback` (or any other fitting name depending on the renames).

Reply:
yes correct, my taught process was that: 
- user should not be using `i18n.localize` function because it is not exposed via uiManager
- when user declares a new Label it feels more natural and less confusing (imo.) if the do `new Label({ text: uiManager.localize('myString')})` instead of  `new Label({ text: uiManager.getLocalizer('myString')})` because what user wants to do is to localize, not to get  `Localizer` to use somewhere else.

again the decision is yours @protyposis , if my thinking is wrong in that regard please lmk. so that I can update it too.

Reply:
Yes `uiManager.localize` feels much better than `uiManager.getLocalizer`, the user shouldn't have to care about implementation technicalities. I'd really like to call it `localize`, but we have another `localize` method - one returns the `Localizer` and one executes it - that is highly confusing. So how about calling this one `uiManager.localize` (it calls `i18n.getLocalizer`) and renaming the "internal" `i18n.localize` into `i18n.performLocalization`?

Reply:
:+1:  yes good idea.